update.rb

```diff
@@ -30,6 +31,12 @@
   "password" => ENV["KIRA_GITLAB_PERSONAL_TOKEN"]
 }
 
+json_credentials = ENV['DEPENDABOT_EXTRA_CREDENTIALS'] || ""
```

Comment:
Please, also fix the docs in the README.

Reply:
I added the environment variabel to the "Per Schedule" Docs ("Global" would also work)